netket/optimizer/qgt/qgt_jacobian_pytree.py

```diff
@@ -55,7 +57,8 @@ def QGTJacobianPyTree(
               models. holomorphic works for any function assuming it's holomorphic
               or real valued.
         holomorphic: a flag to indicate that the function is holomorphic.
-        rescale_shift: If True rescales the diagonal shift
+        rescale_shift: If True rescales the diagonal shift.
+        pdf: |ψ(x)|^2 if exact optimization is being used else None
```

Comment:
Isn't `pdf` assigned inside this function? This makes it look like it can be part of `kwargs`.